cmd/soroban-cli/src/config/secret.rs

```diff
@@ -21,6 +24,10 @@ pub enum Error {
     Ed25519(#[from] ed25519_dalek::SignatureError),
     #[error("Invalid address {0}")]
     InvalidAddress(String),
+    #[error("Ledger does not reveal secret key")]
+    LedgerDoesNotRevealSecretKey,
```

Comment:
Let's not mix ledger implementation into this PR (to reduce total number of code to review)